app/templates/tags.hbs

```diff
@@ -1,28 +1,44 @@
-<section class="gh-view tags-view">
-    <header class="view-header">
+{{#unless selectedTag}}
+<section class="gh-canvas tags-view">
+    <header class="gh-canvas-header">
         {{#gh-view-title}}<span>Tags</span>{{/gh-view-title}}
         <section class="view-actions">
+            <div class="gh-contentfilter gh-btn-group">
+                <button class="gh-btn {{if (eq type "public") "gh-btn-group-selected"}}" {{action "changeType" "public"}}><span>Public tags</span></button>
+                <button class="gh-btn {{if (eq type "internal") "gh-btn-group-selected"}}" {{action "changeType" "internal"}}><span>Internal tags</span></button>
+            </div>
             {{#link-to "tags.new" class="gh-btn gh-btn-green"}}<span>New tag</span>{{/link-to}}
         </section>
     </header>
 
-    {{#gh-tags-management-container tags=tags selectedTag=selectedTag enteredMobile="enteredMobile" leftMobile=(action "leftMobile") as |container|}}
-    <div class="tag-list">
-        <section class="tag-list-content settings-tags {{if tagListFocused 'keyboard-focused'}}">
-            {{#vertical-collection sortedTags
```

Comment:
We need to keep the usage of `{{#vertical-collection}}`, there are too many users with enough tags that the screen will lag really badly without it.